api/nakadi-event-bus-api.yaml

```diff
@@ -548,6 +548,38 @@ paths:
           description: |
             Schema object.
 
+  '/event-types/{name}/cursor-distances':
+    get:
+      tags:
+        - unmanaged-api
+        - monitoring
+        - management-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Calculate the distance between two offsets. This is useful for performing checks for data completeness, when
+        a client has read some batches and wants to be sure that all delivered events have been correctly processed.
+      parameters:
+        - name: X-nakadi-cursor-distances-query
+          in: header
```

Comment:
I would suggest three query parameters for a GET request:

  - partition
  - initial_offset
  - final_offset

This doesn't allow bulk requests, though I dare to question the need for those.
If a bulk request is really needed, put the list of objects into the request body with a POST.